posthog/models/user.py

```diff
@@ -268,6 +269,7 @@ def get_analytics_metadata(self):
             "has_password_set": self.has_usable_password(),
             "has_social_auth": self.social_auth.exists(),  # type: ignore
             "social_providers": list(self.social_auth.values_list("provider", flat=True)),  # type: ignore
+            "instance_url": SITE_URL,
```

Comment:
This would work, though we already have `realm` – would if make sense to do this at `get_instance_realm()` level, where we could return something like "cloud-eu" if `settings.REGION == "EU"`?

Reply:
I think for backward compatibility with our insights I think it's a bad idea to use realm for this? I don't feel super strongly though.

Reply:
Fair enough, I can see how the two Clouds might be the basically same in the context of analytics.